x-pack/plugins/reporting/server/mocks/index.ts

```diff
@@ -5,9 +5,13 @@
  * 2.0.
  */
 
-import { PNG_REPORT_TYPE_V2 } from '../../../common/constants';
+import { ReportingStart } from '../types';
 
-export const metadata = {
-  id: PNG_REPORT_TYPE_V2,
-  name: 'PNG',
+export const reportingMock = {
+  createStart: (): ReportingStart => ({
+    usesUiCapabilities: () => false,
+    registerExportTypes: () => {},
```

Comment:
Thought for a future PR: maybe it would be preferred to **not** have `registerExportTypes` available in our start contract, and force plugins to register their export type in their setup.